developer_manual/scripts/read-config.php

```diff
@@ -1,8 +1,7 @@
 #!/usr/bin/php
 <?php
 /**
- * @author Matthew Setter <matthew@matthewsetter.com>
- *
+ * @author Matthew Setter <matthew@matthewsetter.com> & Martin Mattel <github@diemattels.at>
```

Comment:
I didn't think of this before but assuming that we have more authors than the two of us, let's go with a newline per/author convention. That way it's always simple to read.